app/views/journey/other_journey.scala.html

```diff
@@ -47,15 +47,6 @@
     )
 }
 
-@detailsText = @{
-  val detailsLink = externalLink(
-    text = messages("tariff.declaration.others.linkText.0"),
-    url = appConfig.additionalDeclarationType
```

Comment:
I reckon you can also remove L53 in `conf/externalLinks.conf`. But please check before if the config property is still used somewhere else